src/modules/highlight_follow_streamer_chat/index.js

```diff
@@ -0,0 +1,77 @@
+const settings = require('../../settings');
+const watcher = require('../../watcher');
+const twitch = require('../../utils/twitch');
+const twitchAPI = require('../../utils/twitch-api');
+
+const STYLE_HIGHLIGHT_FOLLOW = 'bttv-highlight-follow-nick';
+const STYLE_HIGHLIGHT_STREAMER = 'bttv-highlight-streamer-nick';
+
+const followsArray = [];
+let currentChannelName;
+let secondRequestAllowed = true;
+
+function putFollowsInArray(direction, limit) {
+    const currentUser = twitch.getCurrentUser();
+    twitchAPI.get(`users/${currentUser.id}/follows/channels/?direction=${direction}&limit=${limit}`)
+        .then(({_total: total, follows}) => {
+            for (let i = 0; i < follows.length; i++) {
+                followsArray.push(follows[i].channel.name);
+            }
+            if (total > 200) {
+                total = 200;
+            }
+            if (total > 100 && secondRequestAllowed) {
+                secondRequestAllowed = false;
+                return putFollowsInArray('asc', total - 100);
+            }
+        });
+}
+
+class HighlightFollowAndStreamerChatModule {
+    constructor() {
+        settings.add({
+            id: 'highlightFollowAndStreamerChat',
+            name: 'Highlight favorite users in the chat',
+            defaultValue: false,
+            description: 'Highlight users you follow and streamer in the chat'
+        });
+        settings.on('changed.highlightFollowAndStreamerChat', () => this.load());
+        watcher.on('chat.message', ($message, messageObj) => this.onMessage($message, messageObj));
+        watcher.on('vod.message', $message => this.onVodMessage($message));
+        watcher.on('load.chat', () => this.load());
+        watcher.on('load.vod', () => this.load());
+    }
+
+    load() {
+        if (settings.get('highlightFollowAndStreamerChat') === false) return;
+        currentChannelName = twitch.getCurrentChannel().name;
```

Comment:
we should make this feature just followed channels, not streamer too .. we can probably assume that if you care about seeing the streamer in chat you'd follow their channel

Reply:
@night Then we can't highlight streamer's nickname with certain CSS. But if you want, I'll do it.